server/pulp/server/db/model/__init__.py

```diff
@@ -700,3 +700,30 @@ class LazyCatalogEntry(Document):
     checksum = StringField()
     checksum_algorithm = StringField(regex=ALG_REGEX)
     data = DictField()
+
+
+class DeferredDownload(Document):
```

Comment:
All documents must inherit from AutoRetryDocument and never from `Document` directly